README.md

```diff
@@ -81,3 +82,27 @@ plug "andreyorst/smarttab.kak" defer smarttab %{
     hook global WinSetOption filetype=(c|cpp) smarttab
 }
 ```
+
+## `autotab` configuration
+
+If you just want to set the behavior based upon your `editorconfig` settings, you can use the `autotab` setting:
+
+```kak
+hook global WinCreate .* %{
+    editorconfig-load
+    autotab
+}
+```
+
+This config will choose `expandtab` or `noexpandtab` based upon the `indent_style` setting as `space` or `tab` respectively.
+If you manually override the `aligntab` option to `false` before running `autotab`, as seen in the below config, this will choose `smarttab` instead of `noexpandtab`:
+
+```kak
+hook global WinCreate .* %{
+    editorconfig-load
+    set-option buffer autotab false
```

Comment:
This might be a typo? `aligntab` is mentioned in the upper paragraph.

Reply:
You're right. Fixed.